parsl/executors/high_throughput/executor.py

```diff
@@ -220,10 +221,14 @@ def _queue_management_worker(self):
         The `None` message is a die request.
         """
         logger.debug("[MTHREAD] queue management worker starting")
+        result_counter = 0
```

Comment:
Is this used for anything?

Reply:
Not anymore, was used for debug. Removed.